qte.sh

```diff
@@ -34,7 +34,7 @@ echo " "
             xcode-select --install
 
             echo "Installing Rosetta
-            sudo softwareupdate --install-rosetta--agree-to-license
```

Comment:
The quotation mark is on the wrong line here 😊 otherwise, good catch! 👍 

echo "Installing Rosetta" <---